pkg/skaffold/event/event.go

```diff
@@ -193,7 +193,7 @@ func DeployInProgress() {
 func DeployFailed(err error) {
 	aiErr := sErrors.ActionableErr(sErrors.Deploy, err)
 	handler.stateLock.Lock()
-	handler.state.DeployState.StatusCode = aiErr.ErrCode
+	handler.state.DeployState.StatusCode = ignoreCancelled(handler.state.DeployState.StatusCode, aiErr.ErrCode)
```

Comment:
`event.DeployFailed` is raised only once in `runner/deploy.go`. Does this case actually happen for deploy too?

Reply:
we will working on adding deploy error codes soon as part of  #4849 and #4692